ccx_messaging/publishers/rule_processing_publisher.py

```diff
@@ -75,6 +75,14 @@ def publish(self, input_msg, response):
                 "RequestId": input_msg.get("request_id"),
             }
 
+            gathered_at = input_msg.get("metadata", {}) \
+                                   .get("custom_metadata", {}) \
+                                   .get("gathering_time", None)
+            if gathered_at:
+                output_msg["Metadata"] = {
```

Comment:
quick comment: checking that `output_msg["Metadata"]` doesn't exist before creating would be more robust. Right now if the "Metadata" was added somewhere else, we would be rewriting it entirely.